### ISSUE TITLE
Resuming partially watched video fails with HTTP 401 Unauthorized

### ISSUE THREAD

**Issue by AlyoshaVasilieva** [TeamPiped/Piped#3151]
### Official Instance

- [X] The bug is reproducible on the [official hosted instance](http://piped.video/), or is API-related.

### Describe the bug

Trying to resume a partially-watched video fails, with HTTP 401 Unauthorized requests spamming logs. Partially-watched video can be viewed by adding `&t=0` to URL so that it starts from the very beginning.

### To Reproduce

1. Enable watch history
2. Open a video such as https://piped.video/watch?v=q1nGvCeDaQQ, seek to some position around the middle
3. Close video
4. Open a new tab with the same video
5. Piped tries to resume the video, but network requests fail.

### Expected behavior

Resuming partially watched video should work again. It used to and I don't remember when this started occurring, but it's been going on for at least a month iirc.

### Logs/Errors

Repeated spam of messages such as

```
       GET https://pipedproxy-yyz.kavin.rocks/videoplayback?expire=1700186079&ei=f3NWZaLWI_uX2_gP64SGoAs&ip=37.19.211.10&id=o-ABIY1CMYIdAcc4Dx-7wTSk28SzL8bgCZwdU3wmaoMGPB&itag=400&source=youtube&requiressl=yes&mh=o3&mm=31%2C26&mn=sn-tt1e7nlz%2Csn-vgqsknzr&ms=au%2Conr&mv=m&mvi=5&pl=24&initcwndbps=2456250&spc=UWF9fw2XK2q2zSmStu_gn-G0czOm2HU&vprv=1&svpuc=1&mime=video%2Fmp4&gir=yes&clen=5817068056&dur=6673.980&lmt=1699302576759607&mt=1700164136&fvip=2&keepalive=yes&fexp=24007246&beids=24350018&c=ANDROID&txp=543G434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=ANLwegAwRQIhAJdDCjTfJ0tEaNynXyXe_cw8baVpem47pr7kHpkgMbGvAiAMMBxCtpjLNeqLtNnUTM-d6JOlQ6XapPB749m-mIKTwg%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AM8Gb2swRgIhAPROgWQvPXtw3_SJp6UIfazFAYKNP0GVXx14I2HiqayMAiEApDt5gp_rL9P0aiT4LUnhv4x0iHnjE7_AHyH3VEYbr5I%3D&cpn=FmC7dLnpncnN4UBk&host=rr5---sn-tt1e7nlz.googlevideo.com&range=227568649-229853910 401 (Unauthorized)
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
C4 @ shaka-player.ui-jLgbSuTI.js:1
qu @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
kE @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
Fe.Ba @ shaka-player.ui-jLgbSuTI.js:1
zp @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
kE @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
Fe.Ba @ shaka-player.ui-jLgbSuTI.js:1
d.request @ shaka-player.ui-jLgbSuTI.js:1
a0 @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
n0 @ shaka-player.ui-jLgbSuTI.js:1
Wl @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
throw @ shaka-player.ui-jLgbSuTI.js:1
n @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
$T @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
throw @ shaka-player.ui-jLgbSuTI.js:1
n @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
$T @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
throw @ shaka-player.ui-jLgbSuTI.js:1
n @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
$T @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
throw @ shaka-player.ui-jLgbSuTI.js:1
n @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
$T @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
throw @ shaka-player.ui-jLgbSuTI.js:1
n @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
$T @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
throw @ shaka-player.ui-jLgbSuTI.js:1
n @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
$T @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
throw @ shaka-player.ui-jLgbSuTI.js:1
n @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
$T @ shaka-player.ui-jLgbSuTI.js:1
qT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
setTimeout (async)
ao.S @ shaka-player.ui-jLgbSuTI.js:1
sn @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
t @ shaka-player.ui-jLgbSuTI.js:1
Promise.then (async)
o @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
Ql @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
YT @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
Yr @ shaka-player.ui-jLgbSuTI.js:1
next @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
H1 @ shaka-player.ui-jLgbSuTI.js:1
C @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
(anonymous) @ shaka-player.ui-jLgbSuTI.js:1
```

Note "401 (Unauthorized)" at end of first long line

### Browser, and OS with Version.

Chrome on Windows 10 x64, also a fresh install of Vivaldi.

### Additional context

Also happens on my private instance.

*Sometimes* it works. I see a bunch of canceled requests, which are usually then followed by a bunch of 401s, but not always.

**Comment by AlyoshaVasilieva** [comment]
The good news is that I don't get 410s anymore after https://github.com/TeamPiped/Piped/commit/f1058b80050dcb0797fd171b0bffdcbc7cf0a05f, however Piped no longer resumes partially-watched videos either. It starts from 0:00.

**Comment by AlyoshaVasilieva** [comment]
Issue is that this is async:

https://github.com/TeamPiped/Piped/blob/6578f0f82bae4181e65485015cea0f4042ba359e/src/components/VideoPlayer.vue#L508-L516

so by the time it sets startTime, startTime was already used (and was 0).

**Comment by FireMasterK** [comment]
Should be fixed with dd8347412677fe26402139ec485707d4a0bc3afe

**Comment by AlyoshaVasilieva** [comment]
Seems to work now, thanks!
